@here/harp-mapview/lib/MapView.ts

```diff
@@ -2581,8 +2582,11 @@ export class MapView extends EventDispatcher {
      * points.
      */
     raycasterFromScreenPoint(x: number, y: number): THREE.Raycaster {
```

Comment:
I think we should move this whole function to the PickHandler, its only used there, and as its still in @alpha should not be used by anybody

Reply:
done